python/ray/resource_spec.py

```diff
@@ -113,8 +113,10 @@ def to_resource_dict(self):
                                  "Received {}.".format(resources))
             if resource_quantity > ray_constants.MAX_RESOURCE_QUANTITY:
                 raise ValueError(
-                    "Resource quantities must be at most {}.".format(
-                        ray_constants.MAX_RESOURCE_QUANTITY))
+                    "Resource quantities must be at most {}. "
```

Comment:
Can we say which resource was requested to be more than the max?

Reply:
Fixed!